bcgov/cas-registration
# Feat/142 facility emission summary

Adds the facility emission summary page & necessary API endpoints/services.

Backend:
- emission_category API endpoint /emission-summary to fetch all the aggregated summary data
- emission_cataegory_service to do the calculations. 
  - Has individual functions to aggregate by emission category
  - Has a special function to determine reporting_only emissions
  - Has a function to return the results of all the individual aggregation functions

Frontend:
- Added a page route for the emission summary
- Added a component to render the summary
- Schema/UI Schema for the read-only form

To test:
- Start a report & save several emissions of data in an activity form
- Go to the `reporting/reports/{version_id}/facilities/{facility_id}/summary` page & see the aggregated results
- You can manipulate the data in the database to change which categories an emission belongs to in order to see the summary display different data
